src/scenario/scenario_events_xml.h

```diff
@@ -0,0 +1,638 @@
+#ifndef SCENARIO_EVENTS_XML_H
+#define SCENARIO_EVENTS_XML_H
+
+#include "building/type.h"
+#include "city/message.h"
+#include "game/resource.h"
+#include "game/settings.h"
+#include "scenario/data.h"
+#include "scenario/scenario_event_data.h"
+
+typedef enum {
+    PARAMETER_TYPE_UNDEFINED = -1,
```

Comment:
I'd set `PARAMETER_TYPE_UNDEFINED` to `0` and increase from there.

It will be easier to handle uninitialized values that way.

Reply:
Done.